src/openrct2/world/Scenery.h

```diff
@@ -228,7 +215,27 @@ enum
     SCENERY_TYPE_PATH_ITEM,
     SCENERY_TYPE_WALL,
     SCENERY_TYPE_LARGE,
-    SCENERY_TYPE_BANNER
+    SCENERY_TYPE_BANNER,
+
+    SCENERY_TYPE_COUNT,
+};
+
+constexpr const int8_t ObjectTypeToSceneryType[OBJECT_TYPE_COUNT] = {
```

Comment:
I think this should be a function instead, you can assert if you're calling from a not scenery object